src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -256,6 +272,38 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods, sites }) => {
           });
         });
 
+        // Creates a new layer
+        const sitesLayer = new google.maps.Data({ map });
+
+        // Loads the objects into the layer
+        sitesLayer.addGeoJson(sites);
+
+        function setSitesStyle(v: boolean) {
+          sitesLayer.setStyle((feature) => {
+            let visible = false;
+            let icon = treeIcon;
+
+            // Only shows sites if there is a tree there
+            if (feature.getProperty('tree_present')) {
+              visible = v && true;
+            }
+
+            // TODO: update this to if the tree was planted within the past three years
+            // If the tree has not been updated within the past three years, use youngTreeIcon
+            const updatedDate = feature.getProperty('updated_at');
+            if (updatedDate < breakpointDate) {
+              icon = youngTreeIcon;
+            }
+
+            return {
+              visible,
+              icon,
+            };
+          });
+        }
+
+        setSitesStyle(false);
```

Comment:
Does this help with performance?

Reply:
This sets the default visibility as false, so when you first load the map you can't see the sites (visibility becomes true after zooming in).